kotsadm/web/src/components/apps/AppDetailPage.jsx

```diff
@@ -296,6 +315,7 @@ class AppDetailPage extends Component {
                         app={app}
                         match={this.props.match}
                         makeCurrentVersion={this.makeCurrentRelease}
+                        makingCurrentReleaseErrMsg={this.state.makingCurrentReleaseErrMsg}
```

Comment:
this is not critical, but the line above maps `makeCurrentRelease` to `makeCurrentVersion `, we should probably follow the same pattern here.:`makingCurrentVersionErrMsg` instead of `makingCurrentReleaseErrMsg`.